index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>The carlot quiz</title>
+
+<link rel="stylesheet" type="text/css" href="quiz.css">
+<link rel="stylesheet" type="text/css" href="/bower_components/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
Typically you'll want to put the bootstrap stylesheet first, so that anything you write in your own .css files doesn't get overwritten by bootstrap.